lib/shoulda/matchers/active_record/have_rich_text_matcher.rb

```diff
@@ -0,0 +1,77 @@
+module Shoulda
+  module Matchers
+    module ActiveRecord
+      # The `have_rich_text` matcher tests usage of the
+      # `has_rich_text` macro.
+      #
+      # #### Example
+      #
+      #     class Post < ActiveRecord
+      #       has_rich_text :content
+      #     end
+      #
+      #     # RSpec
+      #     RSpec.describe Post, type: :model do
+      #       it { is_expected.to have_rich_text(:content) }
+      #     end
+      #
+      #     # Minitest (Shoulda)
+      #     class PostTest < ActiveSupport::TestCase
+      #       should have_rich_text(:content)
+      #     end
+      #
+      # @return [HaveRichText]
+      #
+
+      def have_rich_text(rich_text_attribute)
+        HaveRichText.new(rich_text_attribute)
+      end
+
+      # @private
+      class HaveRichText
+        attr_reader :rich_text_attribute
+
+        def initialize(rich_text_attribute)
+          @rich_text_attribute = rich_text_attribute
+        end
+
+        def description
+          "have :#{rich_text_attribute} as a ActionText::RichText"
+        end
+
+        def failure_message
+          "Expected #{@subject.class} to #{description} but #{@error}"
+        end
+
+        def failure_message_when_negated
+          "Did not expect #{@subject.class} to have a rich text " \
+          ":#{rich_text_attribute}"
+        end
+
+        def matches?(subject)
+          @subject = subject
+          @error = run_checks
+          @error.nil?
+        end
+
+        private
+
+        def run_checks
+          if !has_attribute?
+            ":#{rich_text_attribute} does not exist"
+          elsif !has_expected_action_text?
+            ":#{rich_text_attribute} is a #{@subject.send(rich_text_attribute).class}"
+          end
+        end
+
+        def has_attribute?
+          @subject.respond_to?(rich_text_attribute.to_s)
+        end
+
+        def has_expected_action_text?
+          @subject.send(rich_text_attribute).class == ActionText::RichText
```

Comment:
This can raise `uninitialized constant` if `ActionText::RichText` is not available for the Rails version or available but not loaded.
A solution would be to compare `class.name` with a `'ActionText::RichText'` string

Reply:
True!

I'll update it! Thank you!

Reply:
Done!